homeassistant/components/hive/water_heater.py

```diff
@@ -71,10 +71,23 @@ async def async_setup_entry(
 class HiveWaterHeater(HiveEntity, WaterHeaterEntity):
     """Hive Water Heater Device."""
 
+    def __init__(self, hive, hive_device):                                          
+        """Initialise hive water heater."""                             
+        super().__init__(hive, hive_device)                                      
+        self.current_state = 'Unknown'
+
     _attr_supported_features = WaterHeaterEntityFeature.OPERATION_MODE
     _attr_temperature_unit = UnitOfTemperature.CELSIUS
     _attr_operation_list = SUPPORT_WATER_HEATER
 
+    @property
+    def extra_state_attributes(self):
+        """Return the additional state attributes."""
+        attributes = {}
+        attributes['operation_state'] = self.current_state
```

Comment:
What values can this be?

Reply:
Then this should be a separate binary sensor

Reply:
Thank you for your input @joostlek.  However, (_with the greatest respect_) I struggle to understand how you can make that call without reading the whole discussion.

Reply:
I saw that it was a discussion point, but I don't know why it was one, couldn't really figure that one out